list/defaultitem.go

```diff
@@ -139,11 +149,22 @@ func (d DefaultDelegate) Render(w io.Writer, m Model, index int, item Item) {
 		return
 	}
 
+	if d.ShowDescription {
+		lines := strings.Split(desc, "\n")
+		if len(lines) > d.height-1 {
+			desc = strings.Join(lines[0:d.height-1], "\n")
+		}
+	}
+
 	// Prevent text from exceeding list width
 	if m.width > 0 {
```

Comment:
can we assume this is always >0, if so I can move the if above inside this one and avoid one strings.Split :P